src/libUtils/IPConverter.cpp

```diff
@@ -29,9 +51,45 @@ const std::string IPConverter::ToStrFromNumericalIP(
   return std::string(str);
 }
 
-const boost::multiprecision::uint128_t IPConverter::ToNumericalIPFromStr(
-    const std::string& ipStr) {
-  struct in_addr ip_addr;
-  inet_pton(AF_INET, ipStr.c_str(), &ip_addr);
-  return (boost::multiprecision::uint128_t)ip_addr.s_addr;
+void LogUnsupported(const string& ip) {
+  LogBrandBugReport();
+  cerr << "Error: Unknown address type " << ip << ", unsupported protocol\n"
+       << endl;
+}
+
+void LogInvalidIP(const string& ip) {
+  LogBrandBugReport();
+  cerr << "Error: address " << ip
+       << " does not contain a character string "
+          "representing a valid network address\n"
+       << endl;
+}
+
+void LogInternalErr(const string& ip) {
+  LogBrandBugReport();
+  cerr << "Internal Error: cannot process the input IP address " << ip << ".\n"
+       << std::endl;
+}
+
+int ToNumericalIPFromStr(const std::string& ipStr,
```

Comment:
Might be better to simplify return type to `bool` and we just consider `true` as successful conversion case.

Reply:
Done